src/lib/components/Topology.tsx

```diff
@@ -52,14 +71,21 @@ export class TopologyWrapper extends React.Component<TopologyProps> {
   componentWillUnmount() {
     if (this.paranoid) {
       this.paranoid.destroy();
+      this.paranoid = undefined;
```

Comment:
Is it necessary such removal? Does method `.destroy` not remove instance?

Reply:
It's for safety. This will remove reference to object, so GC could collect it